scripts/dockerfile_fragments.py

```
@@ -13,170 +12,119 @@
+    begin = "#" * 3 + " BEGIN"
+    end = "#" * 3 + " END"
+    with open(dockerfile, "rt") as fp:
+        for line_no, line in enumerate(fp, start=1):
+            for prefix in (begin, end):
+                if line.rstrip().startswith(prefix):
+                    suffix = line[len(prefix) + 1:].rstrip()
+                    if suffix not in replacements:
+                        raise ValueError(
+                            f"Expected replacement for '{prefix} {suffix}' "
+                            f"not found in {dockerfile}:{line_no}"
+                        )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)